jenkins-slaves/jenkins-slave-conftest/Dockerfile

```diff
@@ -1,18 +1,21 @@
-FROM quay.io/openshift/origin-jenkins-agent-base:4.4
+# needed to install the python yq library
```

Comment:
put this after the FROM - i've seen docker/podman crash out if the first line isn't a FROM

Reply:
`done();`